lib/facter/resolvers/networking.rb

```diff
@@ -42,7 +42,7 @@ def clean_up_interfaces_response(response)
 
         def parse_interfaces_response(response)
           parsed_interfaces_data = {}
-          interfaces_data = Hash[*response.split(/^([A-Za-z0-9_]+): /)[1..-1]]
+          interfaces_data = Hash[*response.split(/^([A-Za-z0-9_\.]+): /)[1..-1]]
```

Comment:
This resolver is used only for macOS and FreeBSD but the namespace is not the best. 
For now i think this change is ok as it does not seems to affect macOS.